### ISSUE TITLE
In-game UI enhancements

### ISSUE THREAD

**Issue by Lithrun** [AoTTG-2/AoTTG-2#60]
**How will this enhancement improve the game?**
This issue serves as a continuation of #19. Please await and confirm #33 is done before starting with this issue. This issue is a collection of small UX improvements.

**Describe the solution you'd like**
A complete UI, with additional UX effects

- [ ] Damage "pop" animation. When a player kills a titan, the damage is instantly displayed on the screen. Within AoTTG, an effect would be played where the damage would "pop" into the screen, by making use of scaling. Goes from small to big, and big to small after x seconds.
- [ ] Player Label stabilization. Player labels (their name) are displayed, yet they are attached to the player itself. When a player moves with their head, the label remains in place. Thereby, ensure that the label always stays above the head of the player
- [ ] Block input when typing. When a player is typing in the in-game chat, block the input, so that they for instance won't reload all their blades when they say: "Rrrrrrr". The same applies to other UI elements
- [ ] Chat scrollbar - Add a scrollbar to the ingame chat, and increase the limit of max messages to 100.
- [ ] Chat Limit - Add a limit of 500 characters per message, so that not the entire chat is spammed
- [ ] Pressing spacebar after sending a message activates the chat again, this should not happen
- [ ] Increase font size and reduce the text field size to get more clear text
- [ ] Using the pause menu to exit to the main menu, and join a game again, the pause menu stays disabled. Menu GUI elements should be reset prior to joining
- [ ] Kill feed is tied to the screen height, different resolutions can have the kill feed in the middle of their screen. It should be anchored to the top of the screen

**Comment by Lithrun** [comment]
Issue appears to have too much on it as two previous developers were unable to start working on it. Going to split this up into smaller issues.
